avocado/utils/software_manager.py

```diff
@@ -517,6 +537,95 @@ def provides(self, name):
         else:
             return None
 
+    def info(self, name):
+        """
+        Returns a name for the given package [name]
+
+        :param name: name of the package
+
+        :return full_name: name of the package with version & release
+        """
+        cmd = "%s info %s" % (self.base_command, name)
+        full_name = ""
+
+        try:
+            result = process.system_output(cmd)
+            for line in result.splitlines():
+                if line.startswith('Name'):
+                    full_name += "%s-" % line.split()[-1]
+                if line.startswith('Version'):
+                    full_name += "%s-" % line.split()[-1]
+                if line.startswith('Release'):
+                    full_name += "%s-" % line.split()[-1]
+                    return full_name
+        except process.CmdError, details:
+            log.error(details)
+            return ""
+
+    def build_dep(self, name):
+        """
+        Install build-dependencies for package [name]
+
+        :param name: name of the package
+
+        :return True: If build dependencies are installed properly
+        """
+        option = '-y --tolerant'
+        cmd = 'yum-builddep %s %s' % (option, name)
+
+        try:
+            process.system(cmd, sudo=True)
+            return True
+        except process.CmdError, details:
+            log.error(details)
+            return False
+
+    def source(self, name, dest_path):
+        """
+        Returns a ready-to-build source path for given package [name].
+
+        :param name: name of the package
+        :param dest_path: destination_path
+
+        :return final_dir: path of ready-to-build directory
+        """
+        path = tempfile.mkdtemp(prefix='avocado_')
```

Comment:
it'd be nice to use `prefix='avocado_'  + __name__` to be clear what created this file.

Reply:
... or directly `avocado_software_manager`